pkgs/applications/version-management/git-and-tools/git-machete/default.nix

```diff
@@ -22,19 +22,37 @@ buildPythonApplication rec {
 
   nativeBuildInputs = [ installShellFiles ];
 
-  checkInputs = [ git stestr ];
+  checkInputs = [ git pytest ];
 
+  ### Maybe not needed at all? since `python setup.py test` is executed anyway?
```

Comment:
🤔  @blitz

Reply:
Yes, it looks like the `postCheck` hook can go away.